exe/driver.cc.in

```diff
@@ -42,6 +42,21 @@ void help(po::options_description& desc)
           "Displays its own version string.", desc) << endl;
 }
 
+void conflicting_options(po::variables_map const& vm, string const& opt1, string const& opt2)
+{
+    if (vm.count(opt1) && !vm[opt1].defaulted() && vm.count(opt2) && !vm[opt2].defaulted()) {
+        throw po::error(opt1 + " and " + opt2 + " options conflict: please specify only one.");
+    }
+}
+
+template<typename... TOpts>
+void conflicting_options(po::variables_map const& vm, string const& opt1, string const& opt2, TOpts && ...opts)
+{
```

Comment:
The reduced complexity is nice. I think I'll stick with it for now for the better error message (and I don't think it's that tricky to understand). I wasn't thinking about the counting method, definitely got stuck on comparing every pair of options.